src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -139,7 +140,7 @@ else if( te instanceof IMEChest )
 		{
 			this.myName = GuiText.Chest;
 		}
-		else if( te instanceof PartTerminal )
+		else if( te instanceof AbstractPartTerminal )
```

Comment:
To which class? This case is shared for every terminal and previously all did inherit `PartTerminal`, howerever `AbstractPartTerminal` is now the common superclass for every terminal.
Raising it to `AbstractPartDisplay` makes no sense, as these do not use a terminal like GUI.